ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml

```diff
@@ -341,12 +341,21 @@ tests:
       OPENSTACK_TEST_SKIPS: The OpenStack platform creates Worker nodes in a server
         group\|The OpenStack platform creates Control plane nodes in a server group
       TEST_SKIPS: Services Secondary IP Family \[LinuxOnly\] should function for service
-        endpoints using hostNetwork\| Services Secondary IP Family \[LinuxOnly\] should
-        function for node-Service\| Services Secondary IP Family \[LinuxOnly\] should
-        function for pod-Service
+        endpoints using hostNetwork
```

Comment:
updated to match this list
https://github.com/openshift/release/blob/acc27d4a0e55283f45de7b688dbad1667c1d8f4d/ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml#L125